--- openjpeg-2.0.0/src/lib/openjp2/openjpeg.h.orig	2012-11-19 15:26:08.000000000 +0100
+++ openjpeg-2.0.0/src/lib/openjp2/openjpeg.h	2013-01-07 17:09:24.649324889 +0100
@@ -103,6 +103,8 @@ typedef uint64_t OPJ_UINT64;
 /* Avoid compile-time warning because parameter is not used */
 #define OPJ_ARG_NOT_USED(x) (void)(x)
 
+#include <stdio.h>
+
 /* 
 ==========================================================
    Useful constant definitions
--- openjpeg-2.1.0/src/lib/openjp2/opj_stdint.h~	2014-04-29 09:15:02.000000000 +0200
+++ openjpeg-2.1.0/src/lib/openjp2/opj_stdint.h	2014-11-04 21:22:12.492474675 +0100
@@ -32,6 +32,7 @@
 #define OPJ_STDINT_H
 
 #include "opj_config.h"
+#include <stddef.h>
 #ifdef OPJ_HAVE_STDINT_H
 #include <stdint.h>
 #else
