--- Python-2.7.10/Lib/site.py.orig	2015-08-31 19:08:07.645030705 -0500
+++ Python-2.7.10/Lib/site.py	2015-08-31 19:11:45.761030436 -0500
@@ -525,6 +525,10 @@
 
     abs__file__()
     known_paths = removeduppaths()
+    from sysconfig import is_python_build
+    if is_python_build():
+        from _sysconfigdata import build_time_vars
+        sys.path.append(os.path.join(build_time_vars['abs_builddir'], 'Modules'))
     if ENABLE_USER_SITE is None:
         ENABLE_USER_SITE = check_enableusersite()
     known_paths = addusersitepackages(known_paths)
